travis/clone_oca_dependencies

```diff
@@ -41,20 +44,17 @@ def parse_depfile(depfile, owner='OCA'):
         if not line or line.startswith('#'):
             continue
         parts = line.split()
-        repo = parts[0]
-        if len(parts) > 2:
-            branch = parts[2]
-        else:
+        repo, url, branch, commit = [
+            len(parts) > i and parts[i] for i in range(4)]
```

Comment:
@ebirbe Wouldn't be better to get the len of the parts outside the `for` so `len()`  is not executed in every loop?

Reply:
> Wouldn't be better to get the len of the parts outside the for so len() is not executed in every loop?

Changed, thanks.